src/canisters/frontend/ic-certified-assets/src/tests.rs

```diff
@@ -1166,6 +1235,218 @@ mod allow_raw_access {
     }
 }
 
+#[cfg(test)]
+mod certificate_expression {
+    use crate::http::build_ic_certificate_expression_from_headers_and_encoding;
+
+    use super::*;
+
+    #[test]
+    fn ic_certificate_expression_value_from_headers() {
+        let h = ["a", "b", "c"].to_vec();
+        let c = build_ic_certificate_expression_from_headers_and_encoding(&h, "not identity");
```

Comment:
would be good to have a version that shows what happens when encoding is `identity` (`content-encoding` disappears from the cert expr)

Reply:
test case added